### DISCUSSION TITLE
程式語言學習網站！

### DISCUSSION THREAD

**Discussion by Doreen331** [1101-GS4719-C/Python-Community-GS4719-C#747] [General]
我發現了這個很酷的網站，除了有很多不同語言可以選擇外，其中最特別的是「Try it yourself」的功能，能夠自己動手體會各種不同語法帶來不同的效果，大家可以逛逛這個實用的網站！我個人覺得css的部分很有趣，也學了一些實用的語法，推薦給大家！https://www.w3schools.com/

**Comment by manipulateed** [comment]
網址直接複製貼上即可，不然點開其實連接不到歐(別人傳授的)。

**Reply by chia-kai-chang** [reply]
網址的markdown語法可以參考下圖：
<img width="479" alt="截圖 2022-10-06 下午9 18 21" src="https://user-images.githubusercontent.com/1768891/194323227-c0e922c1-466c-48e7-b852-3e91629aa3e8.png">

另外這邊也有比較詳細的markdown語法教學喔：[感謝學長姐的分享，這是我們這個討論區的第377篇文章喔](https://github.com/NCU-GS4719-Python/Python-Community/discussions/377)

**Comment by HuangHsinTzu** [comment]
感謝分享!!!
最近剛好想複習一下JAVA~

**Comment by AnAnistired** [comment]
感謝分享，十分受用

**Comment by Tuoresa** [comment]
感謝分享！這對於必須同時學習多種程式語言的人來說非常實用！🙏

**Comment by Zoe06060606** [comment]
這個網站真的很實用也易懂

**Comment by Rodrigo-dev-bit** [comment]
[codecademy](https://www.codecademy.com/catalog/language/python)也是一個很不錯的學習網站，之前有用來學java。

**Comment by Pietra1226** [comment]
很棒的網站，也可以體驗到不同語言之間的些微差異，也可以做為入門選擇語言的一個小小參考

**Comment by ching020721** [comment]
謝謝分享，是好操作易了解的網站，對於學習或複習都很有用~

**Comment by a45320614** [comment]
感謝分享~ 這個網站可以幫助許多沒有辦法到學校上課的人學習程式，而且也不用額外花費，是一個非常實用的網站!

**Comment by danwan1027** [comment]
我也常常到這個網站找資料，真的是非常方便

**Comment by williamyen0623** [comment]
每次都在為找不到學習資源而煩惱，感謝分享！

**Comment by CPCpyt** [comment]
感謝分享！讓我這個程式小白在學習的苦海中得到救贖

**Comment by StevenWanglolz** [comment]
我的天啊，各種語言都有! 之後需要自學的話一定會來這邊學習!

**Comment by peggy0329** [comment]
這個網站的UIUX設計的好前顯易懂！
而且直接統整了各式程式語言的學習資源
我真的非常需要這種整合型學習網站
感謝分享！

**Comment by chia-kai-chang** [comment]
我自己在查 `json.loads` 跟 `json.dumps` 語法時也是到W3School網站，它的內容很清楚，就像是程式的「工具書」，而且不會有廣告干擾

**Comment by Evan0979** [comment]
感謝分享,「Try it yourself」的功能強迫學習還滿有用的XD

**Comment by swerfg03joe** [comment]
雖然多數的語法都可以在這個網站中找到，但是過於冷僻的模組或是新的第三方模組還是得去其他地方找

**Comment by WEI-CHEN-CHUN** [comment]
太感謝了，非常實用，想要學習或忘記都可以來看看

**Comment by harrylee1971** [comment]
好猛，有好多程式語言可以用，而且連excel都有欸這樣子就不用買一些外面的書籍來學excel了

**Comment by s96015welson** [comment]
有非常多的程式語言可以使用，而且介面簡潔好上手。不過不是每種語法都有很深入的介紹，所以我認為應該要有點基礎，當需要查詢特定用法時再來使用比較適合。

**Comment by HsuChen88** [comment]
基本上做網站時會邊查邊做，最常用其中一個資訊來源就是w3school了。
